modules/apps/commerce/commerce-pricing-api/src/main/java/com/liferay/commerce/pricing/util/comparator/CommercePricingClassCreateDateComparator.java

```diff
@@ -12,6 +12,7 @@
 /**
  * @author Riccardo Alberti
```

Comment:
Hey @titokenzo, you forgot the deprecated annotation.

Use please that annotation: `@deprecated As of Cavanaugh (7.4.x), with no direct replacement`

Reply:
Hey @titokenzo, those commits `d52dcefaff565efa9b91557d0073f6f79698aff8`, `4806dcc0a1e8b485f79afcc1ce9f5b29d7295ecc`, `b1e2f26a75323e40c64e574f2c8144a507312f65` with breaking change is not correct, please fix it.